conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -860,21 +1011,36 @@ TEST_F(
         power_sustained_Initial.limit *= 2;
 
         if (power_sustained_Max.limitValueLocked == false) {
-          lzt::set_power_limits_ext(power_card_handle, &count_power,
+          status = lzt::set_power_limits_ext(power_card_handle, &count_power,
                                     &power_sustained_Max);
-          auto power_limits_descriptors_getMax =
-              lzt::get_power_limits_ext(power_card_handle, &count_power);
+          if (status == ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
+            continue;
+          }
+          EXPECT_EQ(status, ZE_RESULT_SUCCESS);
+          std::vector<zes_power_limit_ext_desc_t> power_limits_descriptors_getMax;
+          status = lzt::get_power_limits_ext(power_card_handle, &count_power,
```

Comment:
May be we need the UNSUPPORTED_FEATURE check here

Reply:
thanks for the catch added now